vela-templates/internal/cue/task.cue

```diff
@@ -13,10 +13,77 @@ output: {
 				if parameter["cmd"] != _|_ {
 					command: parameter.cmd
 				}
+
+				if parameter["env"] != _|_ {
+					env: parameter.env
+				}
+
+				if parameter["cpu"] != _|_ {
+					resources: {
+						limits:
+							cpu: parameter.cpu
+						requests:
+							cpu: parameter.cpu
+					}
+				}
+
+				if parameter["memory"] != _|_ {
+					resources: {
+						limits:
+							memory: parameter.memory
+						requests:
+							memory: parameter.memory
+					}
+				}
+
+				if parameter["volumes"] != _|_ {
+					volumeMounts: [ for v in parameter.volumes {
```

Comment:
Look, you use `volumeMounts` here instead of `volumemounts`.

Reply:
maybe I mistouch my keyboard...💦